docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations.md

```diff
@@ -224,7 +224,7 @@ This:
 
 Becomes something like this:
 
-```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_queried_table_row_count_to_be.py#L30-L33
```

Comment:
Thank you for catching these. One request to double-check these are still accurate after review / possible revisions :)

Reply:
@Shinnnyshinshin This was a fix to reflect my own changes -- it took me a while to realize what was going on, but then I was happy to be able to draw upon our erstwhile experience with Docusaurus and fix it.  So I did the double checking already by virtue of fixing the line numbers.